src/ui/item.jsx

```diff
@@ -32,6 +33,12 @@ class Item extends React.PureComponent {
       incrementItem(itemName);
     };
 
+    const decrementItemFunc = (event) => {
+      event.preventDefault();
+      
+      decrementItem(itemName);
+    }
```

Comment:
Please fix the linter errors here:
```suggestion
    const decrementItemFunc = (event) => {
      event.preventDefault();

      decrementItem(itemName);
    };
```